distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -2506,6 +2507,68 @@ type Table
+       - attribute_columns: Columns to be used as attributes in the XML.
+       - value_column: Column to be used as the value for the row tag in the XML.
+       - root_name: The name of the root tag in the XML.
+       - row_name: The name of the row tag in the XML.
+       - on_problems: Specifies how to handle warnings if they occur, reporting
+         them as warnings by default.     
+
+       ! Error Conditions
+
+         - If a column in `element_columns`, `attribute_columns` or `value_column` is not in 
+           the input table, a `Missing_Input_Columns` is raised as an error.
+         - If any incomming columns aree not specified in one of `element_columns`, 
+           `attribute_columns` or `value_column`, a `Unused_Columns`
+           is reported according to the `on_problems` setting.
+
+       ? Example to_xml Operation
+
+         Input Table `table`:
+
+            Title                  | Author              | Price | Year 
+           ------------------------+---------------------+-------+------
+            A Tale Of Two Cities   | Charles Dickens     | 9.99  | 1859 
+            The Great Gatsby       | F. Scott Fitzgerald | 5.99  | 1925 
+
+         Result `r = t.to_xml ["Year"] ["Author", "Price"] "Title" "Books" "Book"`:
+
+            <Books>
+              <Book Author="Charles Dickens" Price="9.99">
+                A Tale Of Two Cities
+                <Year>1859</Year>
+              </Book>
+              <Book Author="F. Scott Fitzgerald" Price="5.99">
+                The Great Gatsby
+                <Year>1925</Year>
+              </Book>
+            </Books>
+    @element_columns Widget_Helpers.make_column_name_vector_selector
+    @attribute_columns Widget_Helpers.make_column_name_vector_selector
+    @value_column Widget_Helpers.make_column_name_selector
+    to_xml self (element_columns : (Vector (Integer | Text | Regex) | Text | Integer | Regex) = self.column_names) (attribute_columns : (Vector (Integer | Text | Regex) | Text | Integer | Regex) = []) (value_column : Text | Integer | Regex = "") (root_name : Text = "") (row_name : Text = "") (on_problems : Problem_Behavior = Report_Warning) -> Text =
+        columns_helper = self.columns_helper
+        problem_builder = Problem_Builder.new error_on_missing_columns=True
+        resolved_element_columns = columns_helper.select_columns_helper element_columns Case_Sensitivity.Default False problem_builder
+        java_element_columns = resolved_element_columns.map .java_column
+
+        resolved_attribute_columns = columns_helper.select_columns_helper attribute_columns Case_Sensitivity.Default False problem_builder
+        java_attribute_column = resolved_attribute_columns.map .java_column
+        
+        value_column_for_helper = if value_column != "" then value_column else []
+        resolved_value_column = columns_helper.select_columns_helper value_column_for_helper Case_Sensitivity.Default False problem_builder
```

Comment:
this can be a simple `self.get` for a single column if not `Nothing`.
```suggestion
        resolved_value_column = if value_column.is_nothing then Nothing else self.get value_column
        java_value_column = if resolved_value_column.is_nothing then Nothing else resolved_value_column.java_column
```

Reply:
Made it a self.at to get the missing error message